src/bindings/python/setup.py

```diff
@@ -43,6 +43,7 @@
     "openvino.frontend",
     "openvino.preprocess",
     "openvino.offline_transformations",
+    "openvino._offline_transformations",
```

Comment:
Why do we need this?

Reply:
Refer to discussion in the ticket (attached to the description).